karpenter/datadog_checks/karpenter/metrics.py

```diff
@@ -126,6 +126,28 @@
     'workqueue_retries': 'workqueue_retries',
     'workqueue_unfinished_work_seconds': 'workqueue.unfinished.work_seconds',
     'workqueue_work_duration_seconds': 'workqueue.work.duration_seconds',
+    # new Karpenter V1 metric names
+    'karpenter_nodes_termination_duration_seconds': 'nodes.termination.time_seconds',
+    'karpenter_nodes_terminated_total': 'nodes.terminated',
+    'karpenter_nodes_leases_deleted_total': 'nodes.leases_deleted',
+    'karpenter_nodes_created_total': 'nodes.created',
+    'karpenter_pods_startup_duration_seconds': 'pods.startup.time_seconds',
+    'karpenter_voluntary_disruption_queue_failures_total': 'disruption.replacement.nodeclaim.failures',
```

Comment:
```suggestion
    'karpenter_voluntary_disruption_queue_failures': 'disruption.replacement.nodeclaim.failures',
```
we ignore `_total` for counters